controllers/mondooauditconfig_controller.go

```diff
@@ -23,15 +23,16 @@ import (
 
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"go.mondoo.com/mondoo-operator/api/v1alpha1"
+	mondoov1alpha1 "go.mondoo.com/mondoo-operator/api/v1alpha1"
```

Comment:
It looks like this file already was importing this package as `v1alpha`. Just s/mondoov1alpha1/v1alpha1/ in your new lines of code, and you should be able to get rid of importing the same package twice here.
```suggestion
	"go.mondoo.com/mondoo-operator/api/v1alpha1"
```